### PR TITLE
#420 Info Badge After Activating A Repo.

### PR DESCRIPTION
**Pull request by criske**
pr for #420

Self XDSD

### CODE REVIEW

**Comment by zoeself** [comment]
@criske thank you for your Pull Request. I'll assign someone to review it soon.

**Comment by criske** [comment]
@amihaiemil Unless I'm missing something, this puzzle was way overestimated on time. It took me at most 20 min to complete.

**Comment by zoeself** [comment]
@amihaiemil I couldn't find any assignee for this task. This is either because there are no contributors with role ``REV`` available or because the project does not have enough funds.

Please, make sure there is at least one available contributor with the required role and the project can afford to pay them.

File: src/main/resources/templates/project.html [hunk 1]
```diff
@@ -903,7 +906,7 @@ <h3>This repository is not active</h3>
                                 data: form.serialize(),
                                 success: function (project) {
                                     $("#loadingProject").hide();
-                                    displayProject(project.selfOwner, project);
+                                    displayProject(project.selfOwner, project, true);
```
Comment (amihaiemil) [inline src/main/resources/templates/project.html:909]: @criske is this the only place where we change the function call? Shouldn't we also have ``displayProject(project.selfOwner, project, false)`` somewhere? :D

**Review by amihaiemil — COMMENTED** [review]
@criske just one question

**Comment by amihaiemil** [comment]
@criske 

> Unless I'm missing something, this puzzle was way overestimated on time. It took me at most 20 min to complete.

No, this is everything. I'll change the ticket estimation then :D

File: src/main/resources/templates/project.html (hunk 1 above)
Comment (criske) [reply src/main/resources/templates/project.html:909]: @amihaiemil In our case we ca ignore it when is false; `pmInvitePending` is evaluated as falsy when is undefined and used in the if statement `project.provider === 'github' && pmInvitePending`.

Comment (amihaiemil) [reply src/main/resources/templates/project.html:909]: @criske great :D

**Pull request merged**